docs/app/cloud/secrets/secrets_cli.md

```diff
@@ -11,7 +11,7 @@ If you don't already have Devbox installed, see our [Quickstart](https://www.jet
 
 You can authenticate with Jetify Cloud by running `devbox secrets auth login`. This will launch the browser authentication flow to sign into Jetify Cloud.
```

Comment:
```suggestion
You can authenticate with Jetify Cloud by running `devbox auth login`. This will launch the browser authentication flow to sign into Jetify Cloud.
```